lib/rails/generators/sessionstore/dynamodb/dynamodb_generator.rb

```diff
@@ -46,9 +40,9 @@ def file_name
         name.underscore
       end
 
-      # @return [String] migration version using time stamp YYYYMMDDHHSS
-      def self.next_migration_number(dir = nil)
-        Time.now.utc.strftime("%Y%m%d%H%M%S")
+      # @return [String] activerecord migration version
+      def migration_version
```

Comment:
Is this being used? I dont see it in code - or is it used by Rail's somewhere?

Reply:
It's used in the template `sessionstore_migration.rb`. The migrations are versioned after Rails 5 and inherit a different base class.